docs/guidelines/platforms/android/guideline_percievable_android.md

```diff
@@ -219,31 +294,42 @@ If the app you are building provides multi-dimensional information, it is recomm
 </ConstraintLayout>
 ```
 
-- Headings within text
-
-It is also possible to use _headings_ to summarize groups of text that appear on the screen. To mark a view to be treated as a heading, you can set `android:accessibilityHeading` to true.
+In Compose, nested groups can be merged in a similar way
 
-That way, users of accessibility services can choose to navigate between headings instead of navigating between paragraphs or between words which can improve the text navigation experience.
+```kotlin
+@Composable
+fun FestivalEventTable() {
+    Row {
+        Column(modifier = Modifier.semantics(mergeDescendants = true) { }) {
+            // UI elements that describe the events on Stage A
+        }
+
+        Column(modifier = Modifier.semantics(mergeDescendants = true) { }) {
+            // UI elements that describe the events on Stage B
+        }
+    }
+}
+```
 
-:no_entry_sign: **Failure criteria**
+🚫 **Failure criteria**
 
 - It is not clear which parts of the screen are contextually connected.
 
 ---
 
-### Meaningful sequence
+### Meaningful sequence (WCAG 1.3.2 - Level A)
 
-*This guideline covers point 1.3.2 Meaningful sequence - Level A of the WCAG standard.*
+If the order of the content displayed on the screen is crucial for understanding, it is very important to make sure it will be presented in the same order when using accessibility services. That way, users of accessibility services (such as TalkBack) can get a clearer picture of the content and possible actions on the current screen that is being navigated through.
 
-:white_check_mark: **Success criteria**
+✅ **Success criteria**
 
-If the order of the content displayed on the screen is crucial for understanding, it is very important to make sure it will be presented in the same order when using accessibility services. That way, users of accessibility services (such as TalkBack) can get a clearer picture of the content and possible actions on the current screen that is being navigated through.
+By default, screen elements are read out from the top left to the bottom right. This is a standard way in which nearly all screens work correctly in most scenarios. In the occasions where that is not sufficient, a full-screen redesign with the correct logical and traversal order should be considered. If that is not an option, the traversal order can be modified manually.
 
-The best solution should be a full-screen redesign with the correct and logical traversal order. If that is not an option, on apps with minSdk >= 22, setting `android:accessibilityTraversalBefore` or `android:accessibilityTraversalAfter` attributes could help create a new order of displayed views that will make a bit more sense to TalkBack users.
+On apps with minSdk >= 22, setting `android:accessibilityTraversalBefore` or `android:accessibilityTraversalAfter` attributes could help create a new order of displayed views that will make a bit more sense to TalkBack users.
```

Comment:
In my scenario I had problems adjusting the traversal order of `ComposeView` and `FAB`, and the solution was to set the traversal other through `AccessibilityDelegateCompat`. I haven't found any similar reports on the web, so I can't say that it's a known issue. However, mentioning this as one of the options seems like a good idea 😄 

```
ViewCompat.setAccessibilityDelegate(
        bottomBarComposeView,
        object : AccessibilityDelegateCompat() {
            override fun onInitializeAccessibilityNodeInfo(host: View, info: AccessibilityNodeInfoCompat) {
                super.onInitializeAccessibilityNodeInfo(host, info)
                info.setTraversalAfter(fab)
            }
        },
)
```

Reply:
Thank you! This does work generally, or at least it did in the very simple example that we have, so I think it's good to leave it as a simpler option. I do hope that it works more often than not, but who knows with Google. :D

Since we already have a similar code snippet (for setting traversal order programmatically when minSdk < 22), I just added a small disclaimer, so people are aware to test it before concluding it works. 
The disclaimer is general, but I still wanted to mention `ComposeView` specifically as it seems to be the main culprit in this case and I think it's good to be careful with it.